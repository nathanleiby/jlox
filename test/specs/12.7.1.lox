class FirstIniter {
  init(a) {
    this.a = a;
  }

  getA() {
    return this.a;
  }
}

var fi = FirstIniter("hello 12.7.1");
print fi.getA();
