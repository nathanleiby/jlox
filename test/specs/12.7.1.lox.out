hello 12.7.1
